tests/conftest.py

```diff
@@ -43,6 +44,21 @@ def build(self, **kwargs):
 
     return BillFactory()
 
+@pytest.fixture
+@pytest.mark.django_db
+def legislative_session(db):
```

Comment:
i'm totally fine with this fixture and don't require any changes! just a question for science: is there a reason you didn't use the factory pattern you've used elsewhere for this particular fixture?

Reply:
"for science"...heh.

We use factory-style fixtures whenever we instantiate more than one instance of a model within a single test, [e.g., here](https://github.com/datamade/la-metro-councilmatic/blob/master/tests/test_bills.py#L26). The tests for this PR do not require that of legislation sessions, and until the tests do, then I think we can leave the `legislative_session` fixture as a simple one.